autogpt_platform/frontend/src/components/integrations/credentials-provider.tsx

```diff
@@ -46,6 +46,7 @@ const providerDisplayNames: Record<CredentialsProviderName, string> = {
   smartlead: "SmartLead",
   smtp: "SMTP",
   reddit: "Reddit",
+  proxycurl: "Proxycurl",
```

Comment:
Proxycurl or ProxyCurl?
```suggestion
  proxycurl: "ProxyCurl",
```

Reply:
also P comes before R :)

Reply:
https://nubela.co/proxycurl/ - The service is lowercased Proxycurl